packages/client/src/index.test.ts

```diff
@@ -16,9 +16,11 @@ jest.mock('./verify-id-token');
 const DOMAIN = 'logto.dev';
 const BASE_URL = `https://${DOMAIN}`;
 const CLIENT_ID = 'client1';
+const OPENID = 'openid';
+const SCOPE = OPENID;
```

Comment:
feels like no need to define a const SCOPE = OPENID here.  We could define the value whenever it is being used. i.e. new LogtoClient
as scope should be optional param.

1. be default no need to pass in Scope to all the current existing tests. 
2. Create a test does not pass in Scope as check the `client.scope` contains OPENID
3. Create a test pass scope = OPENID, check the `client.scope` contains only one single OPENID
4. Create a test pass scope = `${OPENID} email`, check the `client.scope` contains OPENID and email

should be good enough